<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>酒城美声音频</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/backgroud.jpg);
border-radius:55px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>760</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>【酒城美声音频】www.jcms.com  联系微信：HongKongNiceBoy</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>64</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_minimized">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/img/zuixiaohua.png</normaloff>:/img/zuixiaohua.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_close">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/img/close.png</normaloff>:/img/close.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-family:'Microsoft YaHei';
text-align:top;

font-size:15px;
font-weight:bold;
}</string>
   </property>
   <property name="text">
    <string>www.jcms.com 联系微信：xxx</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>561</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
    border:none;    
    border-bottom:2px solid #B3B3B3;
    font-family:'Microsoft YaHei';
    font-size:18px;
    font-weight:bold;
    }
QLineEdit:hover{
    border-bottom:3px solid #66A3FF;
    }
QLineEdit:focus{
    border-bottom:3px solid #E680BD;
    }</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string>123</string>
      </property>
      <property name="placeholderText">
       <string>请输入授权码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_login">
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family:'Microsoft YaHei';
text-align:top;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #ffd2f0, stop:1 #b0cbf8);
border-radius:10px;
border:none;

font-size:20px;
font-weight:bold;
}
QPushButton:hover
{
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(170, 0,20,255 ), stop:1 rgba(0,0,255, 200));
}</string>
      </property>
      <property name="text">
       <string>一键安装激活Studio One</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>612</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_install">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>调试install</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_activate">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>调试activate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_uninstall">
      <property name="text">
       <string>调试uninstall</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_deactivate">
      <property name="text">
       <string>调试deactivate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_done">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-family:'Microsoft YaHei';
text-align:top;

font-size:20px;
font-weight:bold;
}</string>
   </property>
   <property name="text">
    <string>安装完成</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1149</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>1117</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_minimized</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1108</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
